code/game/atoms.dm

```diff
@@ -1769,3 +1769,27 @@
 /atom/proc/InitializeAIController()
 	if(ai_controller)
 		ai_controller = new ai_controller(src)
+
+/**
+ * Point at an atom
+ *
+ * Moved from pointed() verb in mob.dm
```

Comment:
Don't need historical note comments.

Reply:
Okay this is a little bit outside of my wheelhouse but I'm moving fast and breaking stuff like a start-up maverick